.github/workflows/push_pr.yml

```diff
@@ -25,6 +25,6 @@ jobs:
   compile_and_test:
     uses: apache/flink-connector-shared-utils/.github/workflows/ci.yml@ci_utils
     with:
-      flink_version: 1.16.0
-      flink_url: https://dist.apache.org/repos/dist/release/flink/flink-1.16.0/flink-1.16.0-bin-scala_2.12.tgz
+      flink_version: 1.16.1
+      flink_url: https://dist.apache.org/repos/dist/release/flink/flink-1.16.1/flink-1.16.1-bin-scala_2.12.tgz
```

Comment:
`https://dist.apache.org/repos/dist/release/flink/flink-1.16.0/flink-1.16.0-bin-scala_2.12.tgz` is not available anymore :-|

Reply:
Have a look at https://github.com/apache/flink-connector-jdbc/pull/21 with how we've solved it :)

Reply:
Thanks @MartijnVisser , still needs version change, right? 1.16.0 -> 1.16.1?

Reply:
> Thanks @MartijnVisser , still needs version change, right? 1.16.0 -> 1.16.1?

It depends if you want to support 1.16.1 (I think you would). But if you would like to remain on 1.16.0, at least the link isn't dead anymore :)

Reply:
Gotcha, thank you! (yes, I think we should support 1.16.1)

Reply:
(addressed in https://github.com/apache/flink-connector-opensearch/pull/7)